src/avro_copy.cpp

```diff
@@ -461,6 +467,11 @@ static idx_t PopulateValue(avro_value_t *target, const Value &val) {
 	auto union_value = *target;
 	if (val.IsNull()) {
 		avro_value_set_branch(&union_value, 0, target);
+		auto schema_type = avro_value_get_type(target);
+		if (schema_type != AVRO_NULL) {
+			throw InvalidInputException("Cannot insert NULL to non-nullable field of type %s",
```

Comment:
Here I either need to remove the file, or whatever function catches this error needs to remove the file